#!/usr/bin/python3

from models.base_model import BaseModel  # Commented out the problematic line
